packages/manager/src/features/CloudPulse/Overview/GlobalFilters.tsx

```diff
@@ -109,11 +112,18 @@ export const GlobalFilters = React.memo((props: GlobalFilterProperties) => {
               savePreferences
             />
           </Grid>
-          <Grid display="flex" gap={1} item md={4} sm={5} xs={12}>
-            <CloudPulseTimeRangeSelect
+          <Grid
+            display="flex"
+            gap={1}
+            item
+            justifyContent={'end'}
```

Comment:
```suggestion
            justifyContent="end"
```

when it is a straight string, we don't need curly braces, check and correct in entire PR